### DISCUSSION TITLE
[A-L] (14 Luglio 2023) Esercizio 6

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#397] [Esami A-L]
La seguente formula proposizionale in CNF è soddisfacibile?

$$
    \Large \\{\ \\{\neg p,\ \neg q\\},\ \\{\neg p,\ q\\},\ \\{p,\ \neg q\\},\ \\{p,\ q\\},\ \\{s,\ p\\},\ \\{\neg s,\ p,\ \neg q\\}\ \\}.
$$

Se si risponde “SI” definire un assegnamento che la soddisfa, se si risponde “NO” dimostrare l’insoddisfacibilità usando la regola di Risoluzione.

**Comment by cek753** [comment]
Unsat
